conf/report/report.css

```diff
@@ -71,6 +71,10 @@ table.dataTable tfoot th {
   outline: 2px solid #505050;
 }
 
+.first_row {
```

Comment:
I would use:
```css
.report_footer > tr:first-of-type > .report_summary_header
```
In order to explicitly tell the reader which element it applies to.